Resources/Prototypes/Catalog/Cargo/cargo_vending.yml

```diff
@@ -33,7 +33,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 3000
+  cost: 6460 # Cosmatic Drift
```

Comment:
This is more then Quickthreads which contains quite a bit more stuff.

Reply:
From what I know about this restock, for the cost arbitrage test, since it works for both clothesmate and winterdrobe, it needs to have price = clothesmate clothing price + winterdrobe price. Do correct me if I'm wrong.

Reply:
Oh, that makes sense. Not a big deal anyways, cargo is filthy rich on CD.